pkg/kubelet/volume_host.go

```diff
@@ -270,6 +270,20 @@ func (kvh *kubeletVolumeHost) GetNodeLabels() (map[string]string, error) {
 	return node.Labels, nil
 }
 
+func (kvh *kubeletVolumeHost) GetAttachedVolumes() (map[v1.UniqueVolumeName]string, error) {
```

Comment:
GetAttachedVolumes() were used in actual_state_world for both controller and kubelet side.
how about use a different name like GetAttachedVolumesFromNodeStatus?

Reply:
Fixed.